docs/user-guide/concepts/connectors.md

```diff
@@ -10,28 +33,35 @@ The current supported connectors are:
   There are some known issues so please check the [Edge issues](#edge-issues)
   section below.
 
+**Note:** If you are running Windows 10 [build 14951][wsl-interop] (or
+later) and Windows Subsystem for Linux (WSL), `sonarwhal` will be capable
+of running the browsers installed directly on Windows. If you are a
+user of the stable release of Window, you will need to use at least the
+*Fall Creators Update*.
+
 ## Configuration
 
-The following properties can be customized in your `.sonarwhalrc` file,
-under the `options` property of the `connector` for any of the
-officially supported ones:
+`connector`s can be configured. Maybe you want to do request with
+another `userAgent`, change some of the other defaults, etc. To do
+that, you just have to add a property `options` to your `connector`
+property with the values you want to modify:
+
+```json
+"connector": {
+    "name": "connectorName",
+    "options": {}
+}
+```
+
+The following is the list of shared configurations for all `connector`s:
 
 * `waitFor` time in milliseconds the connector will wait after the site is
   ready before starting the DOM traversing. The default value is `1000`
   milliseconds.
 
-The following is the default configuration:
+The default value is `1000`.
 
-```json
-{
-    "connector": {
-        "name": "chrome|jsdom",
-        "options": {
-            "waitFor": 1000
-        }
-    }
-}
-```
+Depending on the `connector`, more configuration options are available.
```

Comment:
`more configuration options are available.` => `other different configuration may be available` ?